src/OpenSage.Game/Gui/Wnd/Controls/ListBox.cs

```diff
@@ -417,16 +437,33 @@ private ListBoxItemDimension GetItemBounds(in Size proposedSize)
             }
 
             const int horizontalPadding = 3;
-            var availableWidth = proposedSize.Width - ((_parent.ColumnWidths.Length + 1) * horizontalPadding);
+            var availableWidth = proposedSize.Width - ((_parent.Columns + 1) * horizontalPadding);
 
             int calculateColumnWidth(int column)
             {
-                return (int) ((_parent.ColumnWidths[column] / 100.0f) * availableWidth);
+                // Distribute the available size evenly
```

Comment:
As above, I prefer that this calculation is done when creating the ListBox in WindowLoader.